frontend/dashboard/components/RepoList/RepoList.tsx

```diff
@@ -111,14 +119,9 @@ export const RepoList = ({
             id={`fav-repo-${repo.id}`}
             onClick={handleToggleFav}
             label={repo.user_has_starred ? t('dashboard.unstar') : t('dashboard.star')}
-            icon={
-              <i
-                className={cn(classes.favoriteIcon, {
-                  'fa fa-fav-filled': repo.user_has_starred,
-                  'fa fa-fav-outline': !repo.user_has_starred,
-                })}
-              />
-            }
+            icon={ repo.user_has_starred 
+                ? (<StarFillIcon name ="star-fill-icon" className={cn(classes.favoriteIcon)} /> )
+                : ( <StarIcon name="star-icon" className={cn(classes.dropdownIcon)} /> )}
```

Comment:
`classnames` er ikke nødvendig når det bare er én klasse. Disse parentesene rundt komponentene kan også fjernes.
```suggestion
            icon={repo.user_has_starred 
                ? <StarFillIcon name="star-fill-icon" className={classes.favoriteIcon} />
                : <StarIcon name="star-icon" className={classes.dropdownIcon} />}
```